components/ChartWrapper.js

```diff
@@ -12,6 +12,9 @@ export const ChartWrapper = styled.div`
       vertical-align: middle;
     }
   }
+  .apexcharts-toolbar {
+    z-index: 0 !important;
+  }
```

Comment:
Do we need this? In my testing even removing this worked fine. 

```suggestion
```

Reply:
@JoyLubega : Did you test without this one? I am thinking should work find even without this. 🤔